### ISSUE TITLE
zlib1g-dev to resolve /usr/bin/ld: cannot find -lz error

### ISSUE THREAD

**Issue by ustoopia** [Edward-Wu/srt-live-server#86]
This is not so much an issue as it is hopefully useful feeback for other people who run in to the same problem. Today I followed the instructions to install srtliveserver on a fresh Ubuntu 20.04 server. During the make processes I received an error and compiling couldn't complete. 

/usr/bin/ld: cannot find -lz
collect2: error: ld returned 1 exit status
make: *** [Makefile:54: all] Error 1

After googling for a bit I found the solution to fix this issue. Just install zlib1g-dev package. (sudo apt install zlib1g-dev). After this I was able to successfully perform the make process.

**Comment by ghost** [comment]
yes I had same issue , it worked after I installed zlib1g-dev package!

Thanks!

**Comment by Edward-Wu** [comment]
thanks your feedback.
